graphdb/src/main/java/com/tinkerpop/blueprints/impls/orient/OrientBaseGraph.java

```diff
@@ -1738,7 +1738,7 @@ public long countEdges(final String iClassName) {
             "Requested command '%s' must be executed outside active transaction: the transaction will be committed and reopen right after it. To avoid this behavior execute it outside a transaction",
             msg.toString());
       }
-      raw.commit();
+      raw.commit(true);
```

Comment:
Not sure about this, before nested tx support this was relatively safe operation, now we need to commit all nested transactions too, but we can't restart them.